packages/kcmsf/src/pages/teams.tsx

```diff
@@ -187,8 +191,23 @@ export const Teams = () => {
             entry={entry}
           />
         </>
-      ) : (
-        <Loader m="xl" />
+      )}
+      {loading && (
+        <>
+          <Text>ロード中</Text>
+          <Loader m="xl" />
```

Comment:
```suggestion
          <Loader size={40]} />
```
試合一覧ページと揃えたほうがよさそうです。